package.json

```
@@ -66,6 +66,7 @@
     "vscode": "^1.30.0"
   },
   "activationEvents": [
+    "workspaceContains:**/.go",
```

Shouldn't this be:
```suggestion
    "workspaceContains:**/*.go",
```
Otherwise it will only activate, if it finds a literal `.go` file.
I wanted to open a PR myself and this happened to me during testing, when omitting the asterisk in the globbing pattern.

---

🤦‍♂️oops, thanks @ToothlessGear